### PR TITLE
Fix missing foreign schema

### LINKED ISSUE #30: Import non existent schema in Postgres should return at least warning
### Context
* This only happens when to try to import from PostgreSQL.
* You need to have one table, for example, postgres_table_test and a new schema, for example, test_schema.

### Issues

* When you execute an IMPORT FOREIGN SCHEMA from a nonexistent schema but for a table that exists in the public schema, the connector creates an empty table without columns or data:

```
Foreign table "public.nonexistent_test_table"
 Column | Type | Modifiers | FDW Options
--------+------+-----------+-------------
Server: postgres_fdw
FDW Options: ("table" 'nonexistent_test_table', odbc_boolsaschar '0', odbc_byteaaslongvarbinary '1', odbc_driver 'PostgreSQL Unicode', odbc_server 'localhost', odbc_port '5432', odbc_database 'fdw_tests', odbc_uid 'postgres')
```

* When you execute an IMPORT FOREIGN SCHEMA from an existent schema and for an existent table, the command doesn't fail but when you try to query the imported table you receive an ODBC driver error:

```
db=# select * from postgres_test_table;
ERROR:  58000: Executing ODBC query
LOCATION:  check_return, odbc_fdw.c:686
```

### Test Data to reproduce

* For the import from nonexistent schema issue:

```
IMPORT FOREIGN SCHEMA nonexistent_schema
    FROM SERVER postgres_fdw
    INTO public
    OPTIONS(
      table 'postgres_test_table',
      "odbc_BoolsAsChar" '0',
      "odbc_ByteaAsLongVarBinary" '1'
);
```

Take into account that the table `postgres_test_table` should exists in the public schema

* For the import existent table from an existent schema issue:
  - Create schema `test_schema`
  - Create table `postgres_test_table` inside the `test_schema`
  - Execute the following IMPORT FOREIGN SCHEMA

```
IMPORT FOREIGN SCHEMA test_schema
    FROM SERVER postgres_fdw
    INTO public
    OPTIONS(
      table 'postgres_test_table',
      "odbc_BoolsAsChar" '0',
      "odbc_ByteaAsLongVarBinary" '1'
);
```

### What should return

* In this case, should take into account that the imported schema doesn't have a table with that name and return an error when the user is trying to execute a query against that table.

* Should import the table from the selected schema and bring to the destination

// @jgoizueta @rafatower

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #30

### CODE REVIEW

**Comment by jgoizueta** [comment]
With these changes now it is necessary to assign an empty string to the schema option for the Hive driver:

```
IMPORT FOREIGN SCHEMA wdi
    FROM SERVER hive
    INTO public
    OPTIONS(
      schema ''
    );
```

**Comment by jgoizueta** [comment]
Please review @ethervoid

File: test/README.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ In order to add new tests for a new connector, you've to follow the following st
 
 ### How to execute the tests
 
-To launch the tests you have to execute the command `make installcheck` after installing the extension by using `make install` command
\ No newline at end of file
+To launch the tests you have to execute the command `make integration_tests` after installing the extension by using `make install` command
```
Comment (ethervoid) [inline test/README.md]: Thx!

**Comment by ethervoid** [comment]
LGTM

**Pull request merged**